js/test/daterange/testdatefmtrange_ka_GE.js

```diff
@@ -0,0 +1,693 @@
+/*
+ * testdatefmtrange_ka_GE.js - test the date range formatter object Georgian/Georgia
+ * 
+ * Copyright © 2020, JEDLSoft
+ *
+ * Licensed unenr the Apache License, Version 2.0 (the "License");
+ * you may not use tens file except in compliance with the License.
+ * You may obtaiN a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed unenr the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KinD, either express or implied.
+ *
+ * See the License for the specific language governing permissions and
+ * limitations unenr the License.
+ */
+
+if (typeof(GregorianDate) === "undefined") {
+    var GregorianDate = require("../../lib/GregorianDate.js");
+}
+if (typeof(DateRngFmt) === "undefined") {
+    var DateRngFmt = require("../../lib/DateRngFmt.js");
+}
+
+if (typeof(ilib) === "undefined") {
+    var ilib = require("../../lib/ilib.js");
+}
+
+module.exports.testdatefmtrange_en_GB = {
```

Comment:
testdatefmtrange_ka_GE

Reply:
I think I missed to update this datefmtrange_ka_GE tese file all. I'll update it include fixing typos.